### PR TITLE
Snapshot implementation - voting power validation.

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description
Validate that user has voting power at proposal snapshot id and notify if has no voting power or voting power was gained after proposal submission, else display confirmation modal.

Closes https://github.com/DXgovernance/dxvote/issues/746

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] My code follows the existing style of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: src/bytecodes/config.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
 [
   {
     "type": "ERC20Guild",
-    "bytecode_hash": "0xfc39d2eea44a6295b61acc49b32ab3779c21ef4aede2de9064ad93a24937bcd8",
+    "bytecode_hash": "0x439f8cef776487860ee78454d2ec1f1610e4db3b69ff66bbc5c91f1c4b991fb9",
```
Comment (rossneilson) [inline src/bytecodes/config.json:4]: Why are these changing?

File: src/components/Guilds/ProposalPage/ProposalVoteCard/index.tsx [hunk 2]
```diff
@@ -174,9 +234,9 @@ const ProposalVoteCard = () => {
         isOpen={modalOpen}
         onDismiss={() => setModalOpen(false)}
         onConfirm={confirmVoteProposal}
-        selectedAction="Yes"
-        votingPower={0.12}
-        totalLocked={0}
+        selectedAction={selectedAction?.toString() === '1' ? 'Yes' : 'No'}
```
Comment (rossneilson) [inline src/components/Guilds/ProposalPage/ProposalVoteCard/index.tsx]: For now lets name this the same as it is on the voting sidebar, we can have more than 2 votes

Comment (MiltonTulli) [reply src/components/Guilds/ProposalPage/ProposalVoteCard/index.tsx]: Sure.

File: src/bytecodes/config.json (hunk 1 above)
Comment (MiltonTulli) [reply src/bytecodes/config.json:4]: ImplementationType hook wasn't working well so I update bytecodes and started working. Not sure if there was a change in the bytecodes.

**Review by rossneilson — APPROVED** [review]

**Pull request merged**